presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/Tuple.java

```diff
@@ -67,13 +71,19 @@ public int compareTo(Tuple o)
         }
 
         for (int i = 0; i < types.size(); i++) {
-            Object o1 = values.get(i);
-            Object o2 = o.getValues().get(i);
+            Optional<Object> o1 = values.get(i);
+            Optional<Object> o2 = o.getValues().get(i);
```

Comment:
Make this `o.values` to be consistent with the above. It's normal for a `compareTo` to look directly at the passed in object. Otherwise, they both should use the getter.